src/utils.ts

```diff
@@ -87,13 +87,21 @@ export function getPrescriptionDetailsEndpoint(encounterUuid: string) {
   return `${fhirBaseUrl}/MedicationRequest?encounter=${encounterUuid}&_revinclude=MedicationDispense:prescription&_include=MedicationRequest:encounter`;
 }
 
-export function getPrescriptionTableEndpoint(
+export function getPrescriptionTableActiveMedicationRequestsEndpoint(
   pageOffset: number,
   pageSize: number,
   patientSearchTerm: string,
-  status: string
+  date: string
 ) {
-  return `${fhirBaseUrl}/Encounter?_getpagesoffset=${pageOffset}&_count=${pageSize}&subject.name=${patientSearchTerm}&_revinclude=MedicationRequest:encounter&_revinclude:iterate=MedicationDispense:prescription&_has:MedicationRequest:encounter:status=${status}&_sort=-date&_tag=http%3A%2F%2Ffhir.openmrs.org%2Fext%2Fencounter-tag%7Cencounter`;
+  return `${fhirBaseUrl}/Encounter?_query=encountersWithMedicationRequests&_getpagesoffset=${pageOffset}&_count=${pageSize}&patientSearchTerm=${patientSearchTerm}&date=${date}&status=active`;
```

Comment:
Note that we've switched this request and the one below to use the new named query "encountersWithMedicationRequests"